.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,11 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/dotnet",
+  "features": {
+    "ghcr.io/devcontainers/features/dotnet:1": {
```

Comment:
Does adding the dotnet feature bring any benefit when you are already using the dotnet image? If you are not using prebuilds, it probably makes it just a bit slower to start a new Codespace to have this feature specified.

Reply:
This is mainly for external users... they'd only be able to pull packages that are already present in the Azure DevOps feed, so if .NET 6.0 releases an update and one of us hasn't pulled that version in (explicitly or via pipelines) then they'd be unable to restore. Their workaround would be to manually install the .NET 6.0 SDK, or do some NuGet version pinning. 

Unfortunately the way dotnet and NuGet work they will determine what is the "latest" and want to use it, even if a lower version is present and would work. If a version of .NET 6.0 SDK is already installed then this is avoided. This is why my original image was .NET 6.0 and explicitly brought in .NET Core 3.1. The same issue could occur for .NET 3.1 as well, however it's out of support and the latest version is already pulled in (and we're removing it from the repository).